### DISCUSSION TITLE
How to implement http method in keycloak like GET POST PUT

### DISCUSSION THREAD

**Discussion by projectgroup05** [keycloak/keycloak#22021] [Q&A]
Hello Every One 

if any one know about how to implements http methods to differentiate same method like

1. GET 
     /employee/{id}
2. Delete 
    /employee/{Id}
using keycloak latest version 

if any one have springboot reference project let me know

**Comment by danielFesenmeyer** [comment]
It would be helpful, if you could provide some context and describe your use case.
With the current question, we could just guess, but that might not be helpful.
